scripts/ci-ecr-push

```diff
@@ -0,0 +1,76 @@
+#! /bin/bash
+
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the
+# "License"). You may not use this file except in compliance
+#  with the License. A copy of the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and
+# limitations under the License.
+set -e
+
+AWS_REGION="${1:-}"
+AWS_ACCOUNT_ID="${2:-}"
+
+images="registry:2.7.0
+golang:1.15.4
+gcc:7.3.0
+busybox:1.32.0
+amazonlinux:2.0.20200722.0
+amazon/aws-for-fluent-bit:2.7.0"
```

Comment:
can we extract this block to a common file and use `source` here to import it?

Reply:
I think we can also create a function in this common file to retrieve`ECR_URI="${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_REGION}.amazonaws.com"`

Reply:
Ive change the list of images to be sourced from a single file, but personally not a fan of having the ecr uri in a function since it's not likely to ever change and doesnt seem like the deduplication is worth having to jump between files